### ISSUE TITLE
训练问题

### ISSUE THREAD

**Issue by gray5wolf** [wenet-e2e/wespeaker#426]
![Image](https://github.com/user-attachments/assets/8acc5f04-6051-4944-a679-4cd0e96051da)
[ INFO : 2025-06-14 04:05:47,255 ] - |       150|     16300|5.0113e-05|       0.2|    1.1553|    78.579|
[ INFO : 2025-06-14 04:06:05,754 ] - |       150|     16400|5.0099e-05|       0.2|    1.1557|    78.574|
[ INFO : 2025-06-14 04:06:24,230 ] - |       150|     16500|5.0084e-05|       0.2|    1.1558|    78.571|
[ INFO : 2025-06-14 04:06:43,055 ] - |       150|     16600|5.0069e-05|       0.2|    1.1555|    78.575|
[ INFO : 2025-06-14 04:07:01,532 ] - |       150|     16700|5.0054e-05|       0.2|    1.1552|    78.578|
[ INFO : 2025-06-14 04:07:20,020 ] - |       150|     16800|5.0039e-05|       0.2|    1.1551|    78.577|
[ INFO : 2025-06-14 04:07:38,932 ] - |       150|     16900|5.0024e-05|       0.2|    1.1551|    78.578|
[ INFO : 2025-06-14 04:07:57,426 ] - |       150|     17000|5.0009e-05|       0.2|    1.1554|    78.574|
[ INFO : 2025-06-14 04:08:09,053 ] - |       150|     17062|     5e-05|       0.2|    1.1553|    78.578|
[ INFO : 2025-06-14 04:08:09,201 ] - +----------+----------+----------+----------+----------+----------+
我用wespeaker自带的eres2net模型和配置文件进行训练时，为什么150个epoch后acc还不到80

**Comment by cdliang11** [comment]
加了margin，这个训练acc正常。你跑下测试，看看EER指标

**Comment by gray5wolf** [comment]
---- vox1_O_cleaned.kaldi.score -----
EER = 0.920
minDCF (p_target:0.01 c_miss:1 c_fa:1) = 0.085
---- vox1_E_cleaned.kaldi.score -----
EER = 1.069
minDCF (p_target:0.01 c_miss:1 c_fa:1) = 0.117
---- vox1_H_cleaned.kaldi.score -----
EER = 2.027
minDCF (p_target:0.01 c_miss:1 c_fa:1) = 0.197
指标确实还好

**Comment by wsstriving** [comment]
Margin based softmax directly optimize on the angular distribution of embeddings instead of optimizing the final classification cross entropy.
